src/managers/run-manager.js

```diff
@@ -37,7 +37,7 @@
 *       });
 *       rm.getRun()
 *           .then(function(run) {
-*               // the return value of getRun() is an instance of Run Service
+*               // the return value of getRun() is a run object
```

Comment:
if i breakpoint just inside a call to `rm.getRun().then(function(arg1, arg2) {` and use console to get `arguments`, there is only one argument, and it's a run. maybe i'm missing something though ?